templates/4_Post_Clustering.ipynb

```diff
@@ -24,7 +24,8 @@
     "import numpy as np\n",
```

Comment:
<p>This cell needs to be updated. The goal is to show how to update the cell table using the threshold identified in the histogram above. This is still using ECAD</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>